develop/2017-04-29-fabiocorreacordeiro-Local-Transportation.py

```diff
@@ -0,0 +1,568 @@
+# 
+# *§ 1º The expenses established in items I, VII and VIII may be performed by advisors, understood as the effective servants, occupants of special positions or parliamentary secretaries attached to the Chamber of Deputies."*
+# 
+# So, we understanding that expenses with "Fuel and Lubricants" (item IX) can't be used by advisors, only by the congressman.
+
+# Now we will try to find some reimbursements suspects in subquota "Fuel and lubricants". We will use the worst case acceptable to find a limit to expenses, any reimbursement greater than this limit we will consider a suspect reimbursement.
+# 
+# ### We will consider suspecting all reimbursements that exceed the max cost of R$948,90.
+# 
+# The max cost was calculate using the worst case that is possible to fill a gas and lubricant tank of a car. We used the biggest gas tank in Brazilian market (Ford F-250 with 110l tank),
+# the cost of gas in the most expense gas station in Brazil (R4,70) and 10l of most expensive lubricant (R$40). Most of the service stations don't charge to change lubricants.
+# 
+# Fonts: 
+# 
+# Tanks size: https://panelinhanet.wordpress.com/2013/02/20/combustivel-quantos-litros-cabem-no-tanque-do-seu-veiculo/ in 18/mar/2017 
+# 
+# Cost of gas: http://www.anp.gov.br/preco/prc/Resumo_Semanal_Combustiveis.asp in 25/apr/2017 
+# 
+# Lubricant capacity: http://www.autoideia.com.br/capacidade_oleo_motor_automoveis&codmar=ford 
+# 
+# Cost of lubricant: http://www.mercadomineiro.com.br/pesquisa/oleo-lubrificante-pesquisa-precos
+
+# In[101]:
+
+fuel_cost = 4.70
+fuel_capacity = 110
+lubricant_cost = 40
+lubricant_capacity = 10
+max_cost = fuel_cost*fuel_capacity + lubricant_cost*lubricant_capacity
+max_cost
+
+
+# In[102]:
+
+fuel_outliers = fuel[fuel['total_net_value'] > max_cost].sort_values('total_net_value', ascending=False)
+sns.distplot(fuel_outliers['total_net_value'],bins=100)
+plt.title('Reimbursements of "Fuels and lubricants" greater than Max Cost')
+
+
+# In[103]:
+
+fuel_outliers['total_net_value'].sum()
+
+
+# In[104]:
+
+fuel_outliers['total_net_value'].describe()
+
+
+# If we want to be benevolent with the congresspeople we would admit that these reimbursements were the payment of full month, and they agree with the gas station to pay all in only one payment. So let's calculate the max cost of a month.
+# 
+# Using the same logic of worst case, we will consider that the congresspeople drove the same distance that a taxi driver in São Paulo. In São Paulo there is 3.491 taxi drivers work for a taxi company and they drive 20.000.000 km per month. It's about 5,729km per month for each driver.
+# 
+# Consumption of gas of Ford F-250 - 6,93 Km/L
+# 
+# Most expensive cost of gas on Brasília - R$3,88/L
+# 
+# ### The max cost for a full month is R$3,207.

```

Comment:
Would you mind sticking to the en-US standard for decimal and thousand separators? `,` is a thousand separator and `.` is a decimal one. So (line 259 and 261 in this file) it is `6.93` and `3.88` if you use `3,207`, does that make sense?